types/bitcoin-computer/index.d.ts

```diff
@@ -0,0 +1,42 @@
+// Type definitions for @bitcoin-computer/lib 0.9
+// Project: https://github.com/bitcoin-computer/monorepo (Does not have to be to GitHub, but prefer linking to a source code repository rather than to a project website.)
+// Definitions by: jonty007 <https://github.com/jonty007>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+/// <reference types="node" />
```

Comment:
```suggestion
```
it looks there is no direct dependency to Node JS api, so you can skip this reference (this will spare CI additional check when Node types are updated)

Reply:
Super, that's correct. Thank you